client/src/api/Expenses.ts

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+
+type ReqBody = {
+  userId: string,
+  spentAt: string,
+  title: string,
+  amount: number,
+  category: string,
+  note: string,
+}
+
+export function getExpenses() {
+  return axios.get('http://localhost:5000/expenses').then(res => res.data);
```

Comment:
```suggestion
  return axios.get('http://localhost:5000/expenses').then(res => res.data);
```
move 'http://localhost:5000/expenses' this string to constant (also you can check how to do it via object)